src/portable/portable.c

```diff
@@ -940,11 +940,67 @@ static int make_marker_text(const char *image_path, OrderedHashmap *extension_im
         return 0;
 }
 
+static int append_release_log_fields(
+                const PortableMetadata *release,
+                ImageClass type,
+                const char *field_name,
+                char **ret_text) {
```

Comment:
Hmm, this should not be `ret_`, because it's an input-output field. 
Maybe also move that param to the first position.